Formula/navi.rb

```diff
@@ -11,14 +11,18 @@ class Navi < Formula
     sha256 "64b6dac7d4c689ab6aa5c88ef58369249af2d9a6ac44f56769685569a673dd56" => :high_sierra
   end
 
+  depends_on "rust" => :build
   depends_on "fzf"
 
   def install
-    libexec.install Dir["*"]
-    bin.write_exec_script(libexec/"navi")
+    libexec.install Dir["cheats/*"]
+    libexec.install Dir["shell/*"]
+    system "cargo", "install", "--root", prefix, "--path", "."
```

Comment:
We prefer having a `--locked` where possible.

Reply:
+1, indeed

Reply:
Done!